bin/behat-cleanup.sh

```diff
@@ -4,7 +4,7 @@
 # Delete the Pantheon site environment after the Behat test suite has run.
 ###
 
-if ! "$(terminus whoami)"; then
+if terminus whoami > dev/null; then
```

Comment:
This flips the logic of the `terminus whoami` check, meaning that if `terminus whoami` _succeeds_ (assuming an authenticated session), then the `then` condition (for "Terminus unauthenticated") will follow which is the opposite of what you want. You can solve this by simply adding the `!` back.
🔢 
```suggestion
if ! terminus whoami > dev/null; then
```

Reply:
Intentional change. The original check was 
```
terminus whoami > /dev/null
if [ $? -ne 0 ]; then
```
 
so the check should pass if the result is _not_ zero, which is why I removed the `!`.

Reply:
if `terminus whoami` succeeds, it should `exit 0`, meaning an authenticated session.
if `terminus whoami` detects an unauthenticated session, it will `exit 1`.

your `if terminus whoami > /dev/null; then` will evaluate to `true` if the user is _authenticated_ which is the _opposite_ of what you want (and the opposite of what the original test was testing.

Reply:
actually, that's not true either, either way it exits 0.

Reply:
@rwagner00, exit code 0 is a success, any other exit code is a failure. `if ! terminus whoami > /dev/null` is correct. Now if terminus whoami always exits 0, that's a separate problem and this code has never worked.

I wasn't originally going to bring it up in this CR, but generally, I don't believe these checks have any value, as we don't run CI on forks anyway so we could likely remove a lot of this.

Reply:
Resolved in slack